### ISSUE TITLE
Meris editors no longer open in Catalina. Mac Pro 5,1

### ISSUE THREAD

**Issue by chtunes** [ElliotGarbus/Enzo-Editor-and-Librarian#13]
Meris pedal editors no longer open in Catalina 10.15.7. Mac Pro 5,1
Worked beautifully in Mojave.

**Comment by ElliotGarbus** [comment]
Did you do an OS upgrade?
Have you tried to reinstall the editor?

**Comment by ElliotGarbus** [comment]
I just tested the editor on Sonoma, and it works fine.  It is possible that a file was corrupted.  If you provide more information I'll be happy to help you get it working.  Please let me know what is happening and if you are getting any error messages.

When you try and install you may get a security warning.  If this is the issue you need to go into settings to enable the app to be installed.  If this is not the issue, follow the directions below to delete the main.ini file.

1) Delete the main.ini file.  This is a config file that holds some info for the app.  This includes the locations of some files last position and size of the window.  The app will recreate this file if it is missing.  This will not impact any presets.  

- To delete the file: go the finder.  On the menu select go/home
- Select Users
- Select your User folder (It will be under the user directory, with your user name)
- Press command-shift-> this will toggle the hidden files, you want the hidden files visible.
- select the .enzo folder.
- delete the main.ini file
- restart the app.

Let me know if that fixes the issue or if you are continuing to have problems.

**Comment by chtunes** [comment]
Thank you Elliot!
Your software and help have help my productions immensely.

Yes I upgraded from Mojave to Catalina trying three methods till one worked. 
OpenCore Martin Lo, then open Core Patcher and finally  Dosdudes Catalina Patcher.
Dosdudes got me there and I used Apple's migrate app to transfer all my settings.
I made two log in entities in the process. The one with migrated settings has your Meris editors working now based on your 
instructions. The other login (clean install) still does not have your software opening after your instructions and a fresh reinstall. At this point I will use the migrated install log in and get back to work. The other clean install log in still has me perplexed and after all the codes and Terminal entries that I've made I've created a bit of a needle in a haystick.
I'm sure it's a simple fix or mabe one folder that is blocked from needed permissions. I'm a music producer not a code writer and I'm just trying to get back to work:)

Updated boot rom 144.0. The Mac Pro 5,1 mid 2012, 2 x 3.06 GHz 6 core, 96 GB 1333 MHz DDR3  memory, Sonnett McFiver with 2-4T  SAmsung 990's, Sonnet PCI Sata with 2 -4T SAmsung 860's, UAD 4 core PCI card, UAD 4 core Firewire and Sata out to to two 5 bay HD boxes by Sans digital, 128 i/o from 8 Metric Halo ULN8's through ethernet and a bunch of midi outboard. Digital Performer 11.3, Pro Tools Ultimate 23.9.  It's a miracle I've been able to keep all this working.
Best machine Apple ever made. Waiting for the 4M Mac Pro Tower in 25. In the mean time this machine is my ship and I'll try to keep it from leaking. Thanks again for your excellent work and help!
